src/main/java/edu/harvard/iq/dataverse/api/BuiltinUsers.java

```diff
@@ -36,6 +36,13 @@
     private static final Logger logger = Logger.getLogger(BuiltinUsers.class.getName());
 
     private static final String API_KEY_IN_SETTINGS = "BuiltinUsers.KEY";
+    /**
+     * Users have not requested the ability to retrieve their API token using
+     * their email address but we could. Here's the issue for which we are
+     * enabling login via email address:
+     * https://github.com/IQSS/dataverse/issues/2115
+     */
+    public static boolean retrievingApiTokenViaEmailEnabled = false;
```

Comment:
This boolean was helpful when writing integration tests so I'd like to keep it in. I'm not sure if there's actually demand for looking up API tokens via email. I don't really like that we're using query parameters for this operation anyway, since they may be logged. Logging people's password I mean. It's not great. This was developed in #1818.